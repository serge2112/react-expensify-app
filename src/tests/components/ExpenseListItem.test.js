import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('render expense list item component with one expense', () => {
	const wrapper = shallow(<ExpenseListItem {...expenses[1]}/>);
	expect(wrapper).toMatchSnapshot();
});