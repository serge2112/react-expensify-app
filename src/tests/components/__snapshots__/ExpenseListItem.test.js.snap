// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render expense list item component with one expense 1`] = `
<Link
  className="list-item"
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      Rent
    </h3>
    <span
      className="list-item__subtitle"
    >
      Dec 27th, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $1,800.00
     
  </h3>
</Link>
`;
